server/migrations/20210916060457-add-false-to-ghost-address.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async (t) => {
```

Comment:
Why is this migration necessary? The default is for addresses to be `ghost_address = false` and is only set true when being created with the `import discourse` related routes?

Reply:
@jaybe78 Can you confirm this to be correct? `ghost_address = null` should never exist.

Also on my db dump from a few days ago, 0 rows are update when running this migration.

Reply:
It's a boolean. that migration is not necessary